tests/gdx-tests/src/com/badlogic/gdx/tests/TiledMapObjectLoadingTest.java

```diff
@@ -79,13 +82,33 @@ public void render () {
 		shapeRenderer.setColor(Color.BLUE);
 		Gdx.gl20.glLineWidth(2);
 		MapLayer layer = map.getLayers().get("Objects");
+		AnimatedTiledMapTile.updateAnimationBaseTime();
 		for (MapObject mapObject : layer.getObjects()) {
-			if (mapObject instanceof TextureMapObject) {
-				TextureMapObject tmObject = (TextureMapObject)mapObject;
+//			if (mapObject instanceof TextureMapObject) {
```

Comment:
Funny that you say that, because I wanted to do exactly that. But then a wild @MobiDevelop appeared and forced me to keep it, because people might still be using it. It's not used directly by libgdx anymore, but still part of the general map API.